utils/scapp.h

```diff
@@ -630,18 +641,26 @@ class scApp : public ktxApp {
           "               Disable RDO multithreading (slightly higher compression,\n"
           "               deterministic).\n\n"
           "  --normal_mode\n"
-          "               For ASTC encoder '--encode astc' assumes the input texture is\n"
-          "               a three component linear LDR normal map storing unit length\n"
-          "               normals as (R=X, G=Y, B=Z). The output will be a two component\n"
-          "               X+Y normal map stored as (RGB=X, A=Y), optimized for angular\n"
-          "               error instead of simple PSNR. The Z component can be recovered\n"
-          "               programmatically in shader code by using the equation:\n\n"
+          "               Only valid for linear textures with two or three components.\n"
+          "               If the input texture has three linear components it is assumed to\n"
+          "               be a three component linear normal map storing unit length\n"
+          "               normals as (R=X, G=Y, B=Z). It will be converted to a two \n"
+          "               component X+Y normal map stored as (RGB=X, A=Y) prior to\n"
+          "               encoding.  If the input has 2 linear components it is assumed to\n"
+          "               be an X+Y normal map.\n\n"
+          "               The Z component can be recovered programmatically in shader\n"
+          "               code by using the equations:\n\n"
           "                   nml.xy = texture(...).ga;              // Load in [0,1]\n"
           "                   nml.xy = nml.xy * 2.0 - 1.0;           // Unpack to [-1,1]\n"
```

Comment:
s/normal map/map of unit normals/

Reply:
fixed in https://github.com/KhronosGroup/KTX-Software/pull/493/commits/d92aa03a96ee6fa6b858d8e7315e1b0a61f8e5fb